modules/rest-api/pages/rest-bucket-create.adoc

```diff
@@ -620,6 +626,118 @@ curl -v -X POST http://10.143.201.101:8091/pools/default/buckets/testBucket \
 If successful, the call returns a `200 OK` notification.
 No object is returned.
 
+[#historyretentioncollectiondefault]
+=== historyRetentionCollectionDefault
+
+Whether the default, for all collections within this bucket, is that a record of changes made to documents within the collection should be created and written to disk continuously.
```

Comment:
This sentence doesn't read very well to me - suggest check and redraft?

Reply:
I've changed to: Whether a change-history is made for the bucket.
The change-history records changes made to all collections within the bucket, and is written to disk continuously.
The value can be either `true` (the default) or `false`.